react/index.d.ts

```diff
@@ -606,165 +606,174 @@ declare namespace React {
         onTransitionEndCapture?: TransitionEventHandler;
     }
 
+    // See CSS 3 CSS-wide keywords https://www.w3.org/TR/css3-values/#common-keywords
+    // See CSS 3 Explicit Defaulting https://www.w3.org/TR/css-cascade-3/#defaulting-keywords
+    // "all CSS properties can accept these values"
+    type CSSWideKeywords = "initial" | "inherit" | "unset";
```

Comment:
oh, sorry—one more thing. I don't think the type name should be pluralized. The value `"initial"` is a `CSSWideKeyword`, not a `CSSWideKeywords`. another way of reading this is that a `CSSWideKeyword` is defined as either `"initial"`, `"inherit"`, or `"unset"`.